packages/studio-base/src/panels/ThreeDeeRender/Renderer.ts

```diff
@@ -196,7 +197,10 @@ export class Renderer extends EventEmitter<RendererEvents> implements IRenderer
   #selectedRenderable: PickedRenderable | undefined;
   public colorScheme: "dark" | "light" = "light";
   public modelCache: ModelCache;
-  public transformTree = new TransformTree();
+
+  #transformPool = new ObjectPool(initTransform, 30000);
```

Comment:
```suggestion
  #transformPool = new ObjectPool(initTransform, 30_000);
```

How did you pick 30k? Even if picked randomly add a comment so if we need to revisit this in the future we are not guessing how this number was picked. If there are some considerations at deciding on a "good" number for this note those as well.

Reply:
Added a comment about considerations and also made it a multiple of the max coordinate frame capacity.